template/.github/workflows/keep-alive.yml

```diff
@@ -7,11 +7,8 @@ jobs:
   keep-alive:
     name: Alive
     runs-on: ubuntu-latest
+    permissions:
+      actions: write
     steps:
       - uses: actions/checkout@v4
       - uses: gautamkrishnar/keepalive-workflow@14b7c72e9af14bddbbc1022a6f0bd20b1eac2619
-        with:
-          commit_message: Ah ah ah, stayin' alive
```

Comment:
Why are you removing the config?

Reply:
See https://github.com/gautamkrishnar/keepalive-workflow/releases/tag/2.0.0

v2 is not committing anymore